SH_Script_Extract_Original_Paper_Before_Container-Start/README.md

```diff
@@ -0,0 +1,47 @@
+
+
+\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#
+
+## 	Bash Script to extract a certain file from an ERC    
+######   following the ERC specification of the o2r project
+\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\# 
+
+
+**_Syntax:_**
+
+`./recover_original_paper_from_image.sh $1 $2 [$3]`
+
+Parameters:
+
+*  **$1** : specifies the ID / unique name of an ERC as a String
+   * i.e.:  "NameofERC" / "id"
+
+*  **$2** : pass the location of the ERC within your file system
+   * IF the path includes spaces, it needs to be passed as a String!
+   * i.e.:  "path/to/\ ERC/"
```

Comment:
Relative and absolute paths have been tested and both work. This was not featured in the ReadMe.md as I believed that to be implicit with the use of Bash. It is now clarified to work both ways.